src/technochat/http/ochat.go

```diff
@@ -93,6 +94,10 @@ func (s *Server) ochatInit(r *http.Request) (int, interface{}, error) {
 		}
 	}
 
+	if req.MaxUsers < 2 {
+		req.MaxUsers = 2
```

Comment:
should respond with bad request, not set some default value

Reply:
ok

will be a validation error in future.